driver/src/main/com/mongodb/client/MongoCollection.java

```diff
@@ -262,7 +262,7 @@
      * @mongodb.driver.manual reference/operator/aggregation/changeStream $changeStream
      * @since 3.6
      */
-    ChangeStreamIterable<TDocument> watch();
+    ChangeStreamIterable<Document> watch();
```

Comment:
This should remain TDocument, as with the find method.  fullDocument should be able to use the same decoder as find does by default, no?